cli/cmdlineparser.cpp

```diff
@@ -39,6 +39,9 @@
 #include <list>
 #include <set>
 
+#include <sys/types.h>
```

Comment:
hmm as far as I know this is a posix header so you need to put this in a ifdef.

Reply:
that stackoverflow question you linked to in your comment says that this will work in windows also.. then maybe it's ok to skip the ifdef.. well seems strange to me but if it does work..